src/discussions/comments/data/slices.js

```diff
@@ -16,26 +16,32 @@ const commentsSlice = createSlice({
     commentsById: {
       // Map comment ids to comments.
     },
-    pages: [],
     // Stores the comment being posted in case it needs to be reposted due to network failure.
     // TODO: save in localstorage so user can continue editing?
     commentDraft: null,
-    totalPages: null,
-    totalThreads: null,
     postStatus: RequestStatus.SUCCESSFUL,
+    pagination: {
+    },
   },
   reducers: {
     fetchCommentsRequest: (state) => {
       state.status = RequestStatus.IN_PROGRESS;
     },
     fetchCommentsSuccess: (state, { payload }) => {
       state.status = RequestStatus.SUCCESSFUL;
-      state.pages[payload.page - 1] = payload.ids;
-      state.commentsInThreads = { ...state.commentsInThreads, ...payload.commentsInThreads };
+      Object.keys(payload.commentsInThreads).forEach(threadId => {
```

Comment:
Can't we just use ``payload.ids.forEach`` since that already contains the ids?

Reply:
We can do even shorter - without a loop. What do you think about that? As long as we only request comments for one thread, this should work.
```suggestion
      state.commentsInThreads[payload.threadId] = [
        ...state.commentsInThreads[payload.threadId] || [],
        ...payload.commentsInThreads[payload.threadId] || [],
      ];
```

Reply:
That makes sense. I think we will need to update multiple threads in a single PR, but not comments from multiple threads. So the above is good for me.